code/obj/item/misc_weapons.dm

```diff
@@ -595,16 +605,27 @@ TYPEINFO(/obj/item/sword/pink/angel)
 	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
 	item_state = "ninjaknife"
 	force = 8
-	throwforce = 11
+	throwforce = 18
 	throw_range = 10
 	flags = FPRINT | TABLEPASS | USEDELAY //| NOSHIELD
 	desc = "Like many knives, these can be thrown. Unlike many knives, these are made to be thrown."
+	gang
+		name = "familiar fighting knife"
+		force = 21
```

Comment:
This is a *lot* of damage, more than double that of the switchblade which feels kind of wrong? Also no-one is ever going to throw something that does that much melee damage and that little disorient when thrown.

Reply:
yeah, you're right it's too high when you look at the other knives. i'll knock it down a peg. The main unique trait this brings to the table is definitely the movespeed, given it's in a swathe of other throwing options.